### PR TITLE
Get individual test results

### PR DESCRIPTION
**Pull request by bobbicodes**
Working towards #15, I wrote functions leveraging the [Kaocha](https://github.com/lambdaisland/kaocha) test runner which gives access to the results of all the individual tests.

### CODE REVIEW

**Review by ErikSchierboom — APPROVED** [review]

File: src/exercism/clojure_test_runner.clj [hunk 1]
```diff
@@ -1,14 +1,22 @@
...
-  "Tests a solution file and writes `results.json` to output directory"
-  [slug input output]
-  (spit (str output "results.json")
-        (json/generate-string {:slug slug :input input :output output})))
+(defn -main [slug in-dir out-dir]
+  (let [ns (ns-publics (symbol (str slug "-test")))
+        tests (keys ns)]
+    (shell/sh "cp" (str in-dir (str/replace slug "-" "_") ".clj") "src")
+    (shell/sh "cp" (str in-dir (str/replace slug "-" "_") "_test.clj") "test")
+    (spit (str out-dir "results.json")
```
Comment (ErikSchierboom) [inline src/exercism/clojure_test_runner.clj:15]: Can I see what the `results.json` file is that gets output?

Comment (porkostomus) [reply src/exercism/clojure_test_runner.clj:15]: At this point it just lists each test and whether it passed:

`{"name-bob-test":true,"name-alice-test":true,"two-fer-test":true}`

**Comment by porkostomus** [comment]
I'm actually not very happy with this and am taking a step back to rethink it.

I'm thinking a babashka script should do the trick, it will have near instant startup time and will run on Alpine, no JVM needed.

It includes clojure.test, I just need to figure out how to change its reporting. By default it only outputs the number of failures and errors.

**Comment by porkostomus** [comment]
Superceded by #18

**Pull request closed without merging**